node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/configserver/noderepository/bindings/NodeRepositoryNode.java

```diff
@@ -85,6 +85,8 @@
     public String exclusiveTo;
     @JsonProperty("history")
     public List<Event> history;
+    @JsonProperty("trustStore")
+    public Set<TrustStoreItem> trustStore;
```

Comment:
Sets in a JSON API produce pseudo-random order of elements making unit tests fragile, so consider using List.